packages/block-editor/src/components/block-list/style.scss

```
@@ -702,13 +702,14 @@
 		// Allow clicking through the toolbar holder.
 		pointer-events: none;
 
-		> * {
+		> * > * > * {
```

I debated doing something less specific to nesting levels and more by class.
```suggestion
		.block-editor-block-list__block-selection-button,
		.block-editor-block-contextual-toolbar,
		.block-editor-block-list__empty-block-inserter {
```
That should work and might be better in some way.